js/main.js

```diff
@@ -35,6 +42,18 @@ Site = {
   onResize: function() {
     var _this = this;
 
+    _this.resizeDelay();
+  },
+
+  resizeDelay: function() { 
```

Comment:
doesnt this do the work of a debounce by clearing the timeout before setting it?